### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by allumanatu** [yovanoc/marketplace#1]
Hello Sir,

I don't know much about IT and I am getting that error: 

http://prntscr.com/j3hosu

It says I should contact you. Where is the error?

Kind regards

**Comment by yovanoc** [comment]
Yeah you have to modify your stripe key here too: https://github.com/yovanoc/marketplace/blob/master/resources/assets/js/bootstrap.js

I will add this in the readme thanks
